src/hooks/useRbacPermissions.js

```diff
@@ -22,10 +26,11 @@ const getUserRbacPermissions = (permissions) => {
 
 const useRbacPermissions = () => {
   const chrome = useChrome();
-  const permissions = chrome.getUserPermissions('config-manager');
+  const configManagerPermissions = chrome.getUserPermissions('config-manager');
+  const inventoryPermissions = chrome.getUserPermissions('inventory');
```

Comment:
This is a good idea to use doesHavePermissions since the function has correct handling of wildcards. Though, I would suggest to step even further and use usePermissionsWithContext: https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/doc/RBAC.md. This will integrate nicely with the hooks instead of directly calling chrome.getUserPermissions. You don't even need to fetch available permissions since RBACProvider should already keep the list of account permissions.

Reply:
I will address this in a separate PR.

Reply:
Opened ticket: https://issues.redhat.com/browse/RHINENG-17694